src/app/components/audio-recordings/pages/details/details.component.ts

```diff
@@ -72,41 +71,74 @@ class DetailsComponent extends PageComponent implements OnInit {
   }
 }
 
+// TODO Multiple components required as a hacky bypass to #1711
 @Component({
   selector: "baw-audio-recording-site",
   templateUrl: "./details.component.html",
 })
 class SiteDetailsComponent extends DetailsComponent {}
-
 @Component({
   selector: "baw-audio-recording-point",
   templateUrl: "./details.component.html",
 })
 class PointDetailsComponent extends DetailsComponent {}
+@Component({
+  selector: "baw-audio-recording-region",
+  templateUrl: "./details.component.html",
+})
+class RegionDetailsComponent extends DetailsComponent {}
+@Component({
+  selector: "baw-audio-recording-regions",
+  templateUrl: "./details.component.html",
+})
+class RegionsDetailsComponent extends DetailsComponent {}
+@Component({
+  selector: "baw-audio-recording-project",
+  templateUrl: "./details.component.html",
+})
+class ProjectDetailsComponent extends DetailsComponent {}
+@Component({
+  selector: "baw-audio-recording-projects",
+  templateUrl: "./details.component.html",
+})
+class ProjectsDetailsComponent extends DetailsComponent {}
```

Comment:
OK I've worked it out.... I think... once you add some comments for the routes you expect these components to be served under it will be clearer what you intend